### PR TITLE
Implement rec.frame_slice()

### PR DESCRIPTION
**Pull request by samuelgarcia**

### CODE REVIEW

File: spikeinterface/core/frameslicerecording.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        
+
+    def get_num_samples(self):
+        return self.end_frame - self.start_frame
+
+    def get_traces(self, start_frame, end_frame, channel_indices):
+        if start_frame is None:
+            start_frame = 0
+        if end_frame is None:
+            end_frame = self.get_num_samples()
```
Comment (alejoe91) [inline spikeinterface/core/frameslicerecording.py:56]: @samuelgarcia can you move the parsing of `start_frame` and `end_frame` and also casting to int in the `BaseRecording.get_traces()`?

**Review by alejoe91 — APPROVED** [review]
Looks good :)

File: spikeinterface/core/frameslicerecording.py (hunk 1 above)
Comment (samuelgarcia) [reply spikeinterface/core/frameslicerecording.py:56]: I prefer not.

**Comment by samuelgarcia** [comment]
Can we merge this ?

**Pull request merged**